.gitignore

```diff
@@ -1,8 +1,10 @@
-vendor
-bin
+/vendor
+/.build
+/.release
+/.tarballs
 debug
 *.json
 
 Manifest
-.idea
```

Comment:
Do we not typically ignore these files?

Reply:
No, we don't include editor ignores in our repos. There are a huge variety of these files and it's best practice to configure git with `core.excludesFile` for your development environment.